cli/config/cliconfig.cpp

```diff
@@ -38,3 +60,41 @@ Config CLIConfig::config() const
     return ret;
 }
 
+void CLIConfig::setNetwork(Network const &network) const
+{
+    std::ifstream inFile(m_tomlPath);
+    std::stringstream strStream;
+
+    strStream << inFile.rdbuf();
+    std::string currLine;
+    bool found = false;
+    while (std::getline(strStream, currLine))
+    {
+        if (currLine.find("CLIConfig") != std::string::npos)
```

Comment:
A bit tricky. Cannot use the toml parser (&serializer) here?

Reply:
Unfortunately, not. The library has pretty limited usage, so it does not support changing files... Decided to go this way instead of changing the library.
If, in the future we'll need more changes, I will go with another library

Reply:
In order to change the file = deserialize + change in memory + serialize + save to file.

Reply:
Yes, but there is no native, nor easy support for serialization/deserialization in c++. It would've required too much effort instead of simply doing like this for this sole purpose